mialab/utilities/file_access_utilities.py

```diff
@@ -182,7 +182,8 @@ def __init__(self,
                  file_path_generator: FilePathGenerator,
                  dir_filter: DirectoryFilter = None,
                  file_extension: str = '.nii.gz',
-                 load_pre: bool = False):
+                 load_pre: bool = False,
+                 sanity_check_run: bool = False):
```

Comment:
I've added this flag to be able to iterate quickly and run the pipeline for one train and one test image